assets/js/components/dashboard-splash/dashboard-splash-app.js

```diff
@@ -97,7 +97,7 @@ class DashboardSplashApp extends Component {
 		// Set webpackPublicPath on-the-fly.
 		if ( global._googlesitekitLegacyData && global._googlesitekitLegacyData.publicPath ) {
 			// eslint-disable-next-line no-undef
-			__webpack_public_path__ = global._googlesitekitLegacyData.publicPath;
+			__webpack_public_path__ = global._googlesitekitLegacyData.publicPath; /*eslint camelcase: 0*/
```

Comment:
I think the reason this didn't come up before is that this eslint syntax is turning it off for the whole file (or perhaps just from that point on. We should only be making exceptions per-line.

Can you add `camelcase` after the existing exception for `no-undef` on the line above? 
i.e. `// eslint-disable-next-line no-undef camelcase` should work.

Note other lint warnings/errors may pop up from this change so you may need to add additional single exceptions for them, if any.

Reply:
Used `// eslint-disable-next-line no-undef, camelcase` no other lint errors